service/src/main/resources/capabilities.json

```diff
@@ -0,0 +1,7 @@
+{
+  "submission.limits.maxWorkflows" : 100,
+  "submission.limits.maxInputs" : 200,
+  "submission.limits.maxOutputs" : 300,
+  "workflow.archiving" : true,
+  "workflow.private-workflows.github" : true
```

Comment:
Question for reviewers - should these 2 capabilities be included here considering they have already been released few months back?

Reply:
Hm, that's a good question. Since these are more like features and not configurations, once this is merged in to apps then so too should those features. Any app that may not have these features also won't have this API, so then I think it would be safe enough to omit these two capabilities.

Reply:
Yeah that's what I was thinking too - that since those 2 features have already been merged it doesn't make sense to include in the response currently. I will remove these 2 capabilities from response. In future when new features are added to CBAS maybe then they can be added to capabilities response.